src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+// define functions
+function sortListWithTextValues(list, container) {
+  list.sort((a, b) => a.innerText.localeCompare(b.innerHTML));
+
+  toDocumentAppender(list, container);
+}
+
+function toDocumentAppender(list, container) {
+  list.forEach(elem => {
+    container.append(elem.parentElement);
+  });
+}
+
+function sortNames() {
+  sortListWithTextValues(names, tbody);
+}
+
+function sortPositions() {
+  sortListWithTextValues(positions, tbody);
+}
+
+function sortAges() {
+  ages.sort((a, b) => a.innerText - b.innerHTML);
+
+  toDocumentAppender(ages, tbody);
+}
+
+function sortSalaries() {
+  function getNumberFromSalaryString(str) {
+    return +str.slice(1).split(',').join('');
+  }
+
+  salaries.sort((a, b) => getNumberFromSalaryString(a.innerText)
```

Comment:
it's better to use this as an argument passed to the function

you should be able to move this function to the other file and it should work

Reply:
I use iterable mothod, so it is impossible. Or code length optimization, or performance optimization. Id chosen performance.

Reply:
my comment has nothing in common with the kind of method you use, I recommend you to use salaries from arguments, not from closure